test/unit/lodash.spec.js

```diff
@@ -0,0 +1,568 @@
+
+describe('Lodash training', function () {
+    var params;
+
+    describe('camelCase', function () {
+        before(function () {
+            params = datasets.camelCase();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should convert string to camel case', function () {
+                expect(_.camelCase.apply(_, params)).to.eql('someStringToTestFunction');
+            }
+        );
+    });
+
+    describe('capitalize', function () {
+        before(function () {
+            params = datasets.capitalize();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should convert the first character of string to upper case and the remaining to lower case', function () {
+                expect(_.capitalize.apply(_, params)).to.eql('Some string to test function');
+            }
+        );
+    });
+
+    describe('deburr', function () {
+        before(function () {
+            params = datasets.deburr();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should remove combining diacritical marks from string', function () {
+                expect(_.deburr.apply(_, params)).to.eql('some string to test function');
+            }
+        );
+    });
+
+    describe('endsWith', function () {
+        before(function () {
+            params = datasets.endsWith();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params[1]).to.be.a('string');
+            expect(params).to.have.length.within(2, 3);
+            if (params[2] !== undefined && params.length === 3) {
+                expect(Number.isInteger(params[2])).to.eql(true);
+            }
+        });
+        it('should check if letter on given position is a target string', function () {
+                expect(params[0][params[2] - 1]).to.eql(params[1]);
+            }
+        );
+        it('should check if string ends with the given target string', function () {
+                expect(_.endsWith.apply(_, params)).to.eql(true);
+            }
+        );
+    });
+
+    describe('escape', function () {
+        before(function () {
+            params = datasets.escape();
+        });
+        it('should match types of passing elements', function () {
+            expect(params[0]).to.be.a('string');
+            expect(params).to.have.length(1);
+        });
+        it('should return converted string', function () {
+                expect(_.escape.apply(_, params)).to.eql('Tom &amp; Jerry are old friends');
```

Comment:
In my opinion, without looking to this file, that solution is impossible to reveal